app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
```

Comment:
Be careful here, this should inherit from `ApiController`.
If you have problems with the authentication, check this link out: https://guides.rubyonrails.org/action_controller_overview.html#filters